<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditMainWindow</class>
 <widget class="QMainWindow" name="AuditMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audit your asset</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTreeView" name="sceneTreeView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1093</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="resetAction"/>
    <addaction name="budgetAction"/>
    <addaction name="importBudgetAction"/>
    <addaction name="polyCountAction"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="budgetDock">
   <property name="minimumSize">
    <size>
     <width>89</width>
     <height>111</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Budget and LOD</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTableView" name="budgetTabelView">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="polyCountDock">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Poly Count in Current Scene</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTreeView" name="polyCountTreeView">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="resetAction">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="budgetAction">
   <property name="text">
    <string>Enter your budget</string>
   </property>
  </action>
  <action name="importBudgetAction">
   <property name="text">
    <string>Import Budget</string>
   </property>
  </action>
  <action name="polyCountAction">
   <property name="text">
    <string>Poly Count</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
